tutorials/intro/plot_introduction.py

```diff
@@ -1,404 +1,293 @@
-stc = apply_inverse_raw(raw, inverse_operator, lambda2, method, label,
-                        start, stop)
-
-##############################################################################
-# Save result in stc files:
-
-stc.save('mne_dSPM_raw_inverse_Aud')
-
-##############################################################################
-# What else can you do?
-# ^^^^^^^^^^^^^^^^^^^^^
-#
-#     - detect heart beat QRS component
-#     - detect eye blinks and EOG artifacts
-#     - compute SSP projections to remove ECG or EOG artifacts
-#     - compute Independent Component Analysis (ICA) to remove artifacts or
-#       select latent sources
-#     - estimate noise covariance matrix from Raw and Epochs
-#     - visualize cross-trial response dynamics using epochs images
-#     - compute forward solutions
-#     - estimate power in the source space
-#     - estimate connectivity in sensor and source space
-#     - morph stc from one brain to another for group studies
-#     - compute mass univariate statistics base on custom contrasts
-#     - visualize source estimates
-#     - export raw, epochs, and evoked data to other python data analysis
-#       libraries e.g. pandas
-#     - and many more things ...
-#
-# Want to know more ?
-# ^^^^^^^^^^^^^^^^^^^
-#
-# Browse :ref:`the examples gallery <sphx_glr_auto_examples>`.
-
-print("Done!")
+# Finally, we can estimate the cortical origins of the evoked activity by
+# projecting the sensor data into this subject's :term:`source space`.
+# MNE-Python supports lots of ways of doing this (minimum-norm estimation,
+# dipole fitting, beamformers, etc); here we'll use minimum-norm estimation
+# (MNE) to generate a continuous cortical activation map. To do this we'll
+# need the inverse operator for this subject (the sample data includes one
+# that's been pre-computed), and we'll need to specify a regularization
+# parameter:
+
+inverse_operator_file = os.path.join(sample_data_folder, 'MEG', 'sample',
+                                     'sample_audvis-meg-oct-6-meg-inv.fif')
+inv_operator = mne.minimum_norm.read_inverse_operator(inverse_operator_file)
+# set the regularization parameter (λ²)
+signal_to_noise_ratio = 3.
+regularization_param = 1. / signal_to_noise_ratio ** 2
+# generate the STC
+source_time_course = mne.minimum_norm.apply_inverse(vis_evoked, inv_operator,
+                                                    regularization_param,
+                                                    method='MNE')
+# plot
+source_time_course.plot(initial_time=0.1, hemi='split', views=['lat', 'med'])
+
+##############################################################################
+# The rest of the tutorials have *much more detail* on each of these topics (as
+# well as many other capabilities of MNE-Python not mentioned here:
+# connectivity analysis, encoding/decoding models, lots more visualization
+# options, etc). Read on to learn more!
+
+# sphinx_gallery_thumbnail_number = 9
```

Comment:
This doesn't quite render properly

Reply:
what do you mean?  It's supposed to be rendered as a single comment in a code block with no other commands in it.  (It has to be in a code block for sphinx-gallery to notice it and act accordingly).  I put it at the end (rather than at the beginning as some do) because I think it seems more clearly not part of the lesson when it comes after the closing paragraph.

Reply:
Sorry my bad. I somehow thought that sphinx gallery removes this comment.

Reply:
To me it looks funny / obtrusive the way it's done currently

https://13016-1301584-gh.circle-artifacts.com/0/dev/auto_tutorials/intro/plot_introduction.html#sphx-glr-auto-tutorials-intro-plot-introduction-py

Typically we put these at the top, but if you want to avoid this perhaps it's cleanest to put this in a comment before the line that generates the plot you actually want to use. Then it actually adds a bit of information.

You could also take care of https://github.com/sphinx-gallery/sphinx-gallery/issues/481 if you want :)